glam/api/models.py

```diff
@@ -75,11 +75,38 @@ class Meta(AbstractFenixAggregation.Meta):
         ]
 
 
-class FenixAggregationView(AbstractFenixAggregation):
+class FenixAggregationView(AbstractGleanAggregation):
     class Meta:
         managed = False
         db_table = "view_glam_fenix_aggregation"
 
+class FirefoxAggregation(AbstractGleanAggregation):
```

Comment:
I think FOGAggregation instead of FirefoxAggregation makes it more clear that this is a Firefox on Glean product, instead of Firefox Desktop.

```suggestion
class FOGAggregation(AbstractGleanAggregation):
```

Reply:
made the necessary changes